app/src/main/java/com/rob729/quiethours/QuietHoursApp.kt

```diff
@@ -1,10 +1,15 @@
 package com.rob729.quiethours
 
 import android.app.Application
+import android.app.NotificationManager
+import android.content.Context
+import android.media.AudioManager
 import com.rob729.quiethours.util.StoreSession
 import com.rob729.quiethours.util.WorkManagerHelper
 
 class QuietHoursApp : Application() {
+    val audioManager by lazy { this.getSystemService(Context.AUDIO_SERVICE) as AudioManager }
+    val notificationManager by lazy { this.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager }
```

Comment:
@rob729 I checked again , this line is causing problem. I had tested the app before this change, then it was working fine

Reply:
"Attempt to invoke virtual method 'java.lang.Object android.content.Context.getSystemService(java.lang.String)' on a null object reference"

Reply:
In Utils.kt, define the above variables as lateinit var.
The create a init() function and initalise the variable in it. It will take context as the parameter. Call this function inside the onCreate of application class.

Reply:
It is a similar approach as used in store session

Reply:
ok